packages/eui/scripts/update-versions-log.js

```diff
@@ -3,14 +3,14 @@ const fs = require('fs');
 const { execSync } = require('child_process');
 
 const rootDir = path.resolve(__dirname, '..');
-const versionsLogFile =
-  rootDir + '/src-docs/src/components/guide_page/versions.json';
+const websiteDir = path.resolve(__dirname, '../../website');
+const versionsLogFile = websiteDir + '/static/versions.json';
```

Comment:
```suggestion
const versionsLogFile = path.join(websiteDir, '/static/versions.json');
```

Not that it changes anything in the output (even on Windows which prefers backslashes), but it's a good practice to use `path.join` here :)